docs/diginstroom/sip/2.0/sip_structure/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout:       default
+title:        Structure
+parent:       2.0
+grand_parent:  SIP Specification 
+nav_order:    4
+nav_exclude:  false
+has_children: true
+---
+Editor's Draft
+{: .label .label-yellow }
+# SIP structure
+
+The meemoo SIP consists of a hierarchical directory structure with 2 levels:
+
+- the root directory `/` or _package level_;
+- the `/representations` directory inside the `/` directory or _representation level_.
+
+<figure class="mx-auto">
+  <img src="../../../../../assets/images_spec/sip-structure-2.0.drawio.png" alt="meemoo SIP structure" /> 
+  <figcaption>The directory structure of a meemoo SIP.</figcaption>
+</figure>
```

Comment:
Perhaps it would be better to label the SIP as a whole as 'meemoo SIP' and then be a bit more clear about what constitutes the package level? Now it looks like the package level encompasses everything, whereas it should just be everything except the representations directory and everything in there.